migrations/20181021111949-create-ticket.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
`'use strict'` is used in ES5 to catch weird errors. We are in Node which means we use ES6+, therefore we don't need it at all. To make it easy to remember, every time you have `module.exports` it means ES6+ so no ES5 related stuff should be used. Please read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) about Strict mode